scripts/core/ui/components/Autocomplete.tsx

```diff
@@ -18,6 +19,7 @@ interface IProps<T extends IBaseRestApiResponse> {
 interface IState<T> {
     fetchedItems?: Array<T>;
     loading: boolean;
+    selectedT: T | null;
```

Comment:
what does `T` stand for in `selectedT` ?

Reply:
For the generic object, same as in `Array<T>`

Reply:
is that an idiom? if not I would go for something like `selectedItem`, looks like a typo to me

Reply:
I don't think we use it much. I'll change it :)